docs/guides/actors.md

```diff
@@ -196,12 +196,12 @@ const fetchData = (query) => {
 ```
 
 ::: warning
-It is not recommended to spawn promise actors, as [invoking promises](./communication.md#invoking-promises) is a better pattern for this, since they are dependent on state (self-cancelling) and have more predictable behavior.
+It is not recommended to spawn promise actors, as [invoking promises](./communication.md#invoking-promises) is a better pattern for this since they are dependent on the state (self-canceling) and have more predictable behavior.
```

Comment:
```suggestion
It is not recommended to spawn promise actors, as [invoking promises](./communication.md#invoking-promises) is a better pattern for this since they are dependent on state (self-canceling) and have more predictable behavior.
```

I think "the" is not a good change here. **The** state? Which state is that?